docs/source/providers.rst

```diff
@@ -12,11 +12,23 @@ SQSProvider
 ``SQSProvider`` (located at ``loafer.ext.aws.providers``) receives the following options:
 
     * ``queue_name``: the queue name
-    * ``endpoint_url`` (optional): the base url, only usefull if you don't use AWS
-    * ``use_ssl`` (optional, default=True): SSL usage
     * ``options``: (optional): a ``dict`` with SQS options to retrieve messages.
       Example: ``{'WaitTimeSeconds: 5, 'MaxNumberOfMessages': 5}``
 
+    Also, you might override any of the parameters below from boto library (all optional):
+
+    * ``api_version``
+    * ``aws_access_key_id``
+    * ``aws_secret_access_key``
+    * ``aws_session_token``
+    * ``endpoint_url``
```

Comment:
@wiliamsouza the problem with this is to keep it updated, that's why I've linked boto reference below.